<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PAngleGen</class>
 <widget class="QDialog" name="PAngleGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Angles</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spn_minAngle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spn_MaxAngle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spn_AngleIncs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MinAngle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Angle (°)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MinAngle_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Angle (°)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MinAngle_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of Inclinations</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_cosSpaced">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cos Spaced</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PAngleGen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PAngleGen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
